data/utils/jsoniter/jsoniter.go

```diff
@@ -1,14 +1,30 @@
-// package jsonitere wraps json-iterator/go's Iterator methods with error returns
+// Package jsoniter wraps json-iterator/go's Iterator methods with error returns
 // so linting can catch unchecked errors.
 // The underlying iterator's Error property is returned and not reset.
 // See json-iterator/go for method documentation and additional methods that
 // can be added to this library.
-package jsonitere
+package jsoniter
 
 import (
+	"io"
+
 	j "github.com/json-iterator/go"
 )
 
+const (
+	InvalidValue = j.InvalidValue
+	StringValue  = j.StringValue
+	NumberValue  = j.NumberValue
+	NilValue     = j.NilValue
+	BoolValue    = j.BoolValue
+	ArrayValue   = j.ArrayValue
+	ObjectValue  = j.ObjectValue
+)
+
+var (
+	ConfigDefault = j.ConfigDefault
+)
```

Comment:
since you are mapping some of the things exported in the original package, maybe we should go over the rest of things that `frame_json.go` is using so it doesn't use directly github.com/json-iterator/go?

Reply:
I tried that. But for this PR it'll make things much more complicated. Maybe I'll do it in a separate PR.